### ISSUE TITLE
Specifing input variables for Linearization in Physics/Mechanics

### ISSUE THREAD

**Issue by AlessandroZavoli** [sympy/sympy#26197]
When using the linearization routines in [sympy.physics.mechanics]https://docs.sympy.org/latest/modules/physics/mechanics/linearize.html)
it is not clear how to define the system inputs $r$

In particular, the form
$M \delta x' =  A \delta x + B \delta r$
returns B empty even in the case of applied non-conservative forces that depend on a parameter (i.e., the system input).

What is the correct way to specify that a symbolic variable is a system input?

**Comment by moorepants** [comment]
I don't think you have to do anything. If you have symbols that are functions of time in your equations of motion that are not state variables, then the linearize method treats them as inputs.
